modules/nw-install-config-gateway-api.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-config-gateway-api_{context}"]
+= Installing and configuring gateway API
+
+The following guide provides instructions for using link:https://gateway-api.sigs.k8s.io/[Gateway API] with the Contour Operator on {product-title}.
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `kubectl` command line.
+* You installed the Contour Operator.
+//* Log in as a user with cluster-admin permission.
+
+.Procedure
+
+. Install the Gateway API and dependent resources:
++
+[source,terminal]
+----
+$ kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.13.0/examples/gateway/gateway.yaml
+----
++
+[NOTE]
+====
+Envoy is exposed using a LoadBalancer service. Replace `gateway.yaml` with `gateway-nodeport.yaml` to use a NodePort service instead.
+====
+
+. Verify that all pods in the `projectcontour` namespace are running:
++
+[source,terminal]
+----
+$ kubectl get po -n projectcontour
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                         READY   STATUS      RESTARTS   AGE
+contour-768547cfb8-c2rhn     1/1     Running     0          2m
+contour-768547cfb8-q866f     1/1     Running     0          2m
+contour-certgen-main-rb2h2   0/1     Completed   0          92s
+envoy-d5djm                  2/2     Running     0          2m41s
+envoy-gjwz5                  2/2     Running     0          2m41s
+envoy-hbg6j                  2/2     Running     0          2m41s
+----
++
+The number of Envoy pods depends on how many worker nodes are in your cluster.
++
+[NOTE]
+====
+It can take a few minutes for the Contour Operator to become available.
+====
+
+. Run a test workload:
++
+[source,terminal]
+----
+$ kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.13.0/examples/gateway/kuard/kuard.yaml
+----
+
+. Verify the status of the test workload:
++
+[source,terminal]
+----
+$ kubectl get po,svc,httproute -n projectcontour -l app=kuard
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                         READY   STATUS    RESTARTS   AGE
+pod/kuard-798585497b-9mvwh   1/1     Running   0          5s
+pod/kuard-798585497b-kcjnn   1/1     Running   0          5s
+pod/kuard-798585497b-lnhsn   1/1     Running   0          5s
+
+NAME            TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+service/kuard   ClusterIP   10.96.157.48   <none>        80/TCP    5s
+
+NAME                                  HOSTNAMES
+httproute.networking.x-k8s.io/kuard   ["local.projectcontour.io"]
+----
++
+The application is exposed using an `HTTPRoute` that routes all HTTP requests for `local.projectcontour.io` to service kuard.
+
+. Curl the application hostname:
++
+[source,terminal]
+----
+$ export GATEWAY=$(kubectl -n projectcontour get svc/envoy -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
```

Comment:
__Note:__ Replace `hostname` in the json path with `ip` if your cloud provider uses IP addresses instead of hostnames for load-balancer services.